home-work14/test_for_human.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from human import Human
+
+
+def test_human_grow():
+    human = Human("John", 30, "male")
+    human.grow()
+    assert human.age == 31, "Age is changed to 1"
+
+
+def test_human_change_gender():
+    human = Human("Jane", 25, "female")
+    human.change_gender("male")
+    assert human.gender == "male", "Gender is not changed"
+
+
+def test_human_age_property():
+    human = Human("Sam", 50, "male")
+    assert human.age == 50, "Human age is not 50, parameters parsed wrong"
+    human.grow()
+    assert human.age == 51, "Human does not grow"
+
+
+def test_human_gender_property():
+    human = Human("Alex", 25, "female")
+    assert human.gender == "female", "parameters parsed wrong"
+    human.change_gender("male")
+    assert human.gender == "male", "Gender is not changed"
+
+
+def test_human_grow_1():
+    human = Human("John", 30, "male")
+    human.grow()
+    assert human.age == 31, "Age is not incremented by 1 after calling grow()"
```

Comment:
You already test this function 2 times in previous tests. This one makes no sense